core/modules/css/cssLib.i

```diff
@@ -47,7 +53,118 @@ namespace std {
     %template(StringVector) vector<string>;
 };
 
+%include "sql/SqlErrorObject.h"
+
+%{
+    void setPythonException(const std::string& exceptionStr, const lsst::qserv::css::CssError& ex) {
+        PyObject* module = PyImport_ImportModule("lsst.qserv.css");
+        if (!module)
+            throw;
```

Comment:
I think crashing due to uncaught C++ exception is not very nice, it's better to raise Python exception. You can just do `return` instead of `throw` in this case because import had already set Python error/exception state, so people will see "Import error: ... not found" or something like this.